workers/ohsome_quality_analyst/cli.py

```diff
@@ -141,18 +143,14 @@ def create_indicator(
                     force=force,
                 )
             )
-            if indicator.data is not None:
-                feature["properties"].update(vars(indicator.data))
-            feature["properties"].update(vars(indicator.metadata))
-            feature["properties"].update(vars(indicator.result))
+            feature = update_features_indicator(feature, indicator)
         if outfile is None:
+            # the outfile will be written in the same dir as the infile. the original
+            # name will be kept, but the name of the indicator will be added to it
             outfile = infile.stem + "_" + indicator_name + infile.suffix
```

Comment:
Hey @hn437 @matthiasschaub ,
this is not working for me as expected. When I run the cli the output file will be written to my current working directory, but the location of the input file.

```
:~/GIT_REPOS/ohsome-quality-analyst/workers$ oqt create-indicator -i GhsPopComparisonBuildings -l building_count --infile=/home/h9b/Downloads/test_regions_oqt.geojson
(ohsome-quality-analyst-eBugwxcL-py3.8) h9b@h9b-ThinkPad-T480s:~/GIT_REPOS/ohsome-quality-analyst/workers$ 

ls
Dockerfile  Dockerfile.continuous-integration  ohsome_quality_analyst  poetry.lock  pyproject.toml  scripts  setup.cfg  test_regions_oqt_GhsPopComparisonBuildings.geojson  tests

```

Reply:
it worked for me before I used the helper functions. maybe something was corrupted by using them, I'll look into it on monday. thank you for letting me know!

Reply:
I adressed it in the latest commit. Now, it works for me on Windows, maybe you could check it for linux @Hagellach37 and let me know if it works there as well? Thank you!